CHANGELOG.txt

```diff
@@ -2,6 +2,13 @@
 MABWiser CHANGELOG
 =====================
 
+-------------------------------------------------------------------------------
+April, 15, 2021 1.11.2
+-------------------------------------------------------------------------------
+minor:
+- Updated dtype from np.int to int in base_mab.py line 294 to resolve deprecation warning from numpy>=1.20.0
+- Updated tests to reflect changes in KMeans update in sklearn>=0.24.0
```

Comment:
The change is to the sklearn version dependency, the unit tests are incidental to that. Suggested rephrasing: "Updated sklearn version dependency to >=0.24.0 because of changes in the KMeans implementation"

Reply:
thank you, updated.

Reply:
@skadio I will check all other libraries as well